apps/teams-test-app/e2e-test-data/messageChannels.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "Message Channels",
+  "checkIsSupported": {
+    "capabilityName": "MessageChannels",
+    "expectedOutput": "MessageChannels is not supported",
+    "testUrlParams": []
```

Comment:
This line needs to be removed because it isn't supported in support check.

Reply:
Had to remove the whole block here, but when I add this back in the next PR I'll exclude the testUrlParams section since I don't need to set anything here.